app/assets/stylesheets/application.css.scss

```diff
@@ -278,11 +278,17 @@ main {
     top: 9px;
     right: 2px;
     position: absolute;
-    border-radius: 5px;
     color: white;
+    border-radius: 30px;
     background-color: black;
     border: none;
 }
+.finalize {
+  color: white;
+  border-radius: 30px;
+  background-color: black;
```

Comment:
Color `black` should be written in hexadecimal form as `#000000`<br>Color literals like `black` should only be used in variable declarations; they should be referred to via variable everywhere else.